Atomizer/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.2.6.RELEASE</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>senghout.github.com</groupId>
+	<artifactId>atomizer</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>atomizer</name>
+	<description>Service that handles mapping long URL to short URL</description>
+
+	<properties>
+		<java.version>1.8</java.version>
+		<spring-cloud.version>Hoxton.SR4</spring-cloud.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-netflix-eureka-client</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-netflix-eureka-server</artifactId>
+		</dependency>
```

Comment:
reminds me again why we need to include this since Atomizer is just a client

Reply:
I was just checking to see if you actually read my PR's.

JK. Good point, I was just following the tutorial and it also included it. It did strike me as odd.

Reply:
Looks like it's not needed and there's a bug in the tutorial. Updating.

Reply:
So you're admitting i was right you hoe lol #FeelsGood #Blessed

Reply:
You review your co-workers with that mouth?

Reply:
okay i take it back my bad. i forgot this is public.